internal/pkg/plugin/trello/helper.go

```diff
@@ -0,0 +1,60 @@
+package trello
+
+import (
+	"fmt"
+
+	"github.com/mitchellh/mapstructure"
+	
+	"github.com/merico-dev/stream/pkg/util/log"
+	"github.com/merico-dev/stream/pkg/util/trello"
+)
+
+func buildState(options *Options, ti *TrelloItemId) map[string]interface{} {
+	res := make(map[string]interface{})
+	output := make(map[string]interface{})
+	res["boardId"] = ti.boardId
+	res["todoListId"] = ti.todoListId
+	res["doingListId"] = ti.doingListId
+	res["doneListId"] = ti.doneListId
+
+	output["bid"] = ti.boardId
```

Comment:
"bid" is too simple to show its real meaning. How about boardId/boardID? Pay attention to upper/lowercase. todoid->todoId, doingid->doingId, etc.